### ISSUE TITLE
64 bit port

### ISSUE THREAD

**Issue by blakemcbride** [barak/oaklisp#5]
While Oaklisp works well as a 32 bit program in a 64 bit environment, it would be great to have a true 64 bit port.

**Comment by blakemcbride** [comment]
This is becoming an increasing problem as 32-bit environments are disappearing.  In the not-too-distant future, you may not be able to build 32-bit programs.

**Comment by barak** [comment]
Yes. It isn't too hard in principle, just a matter of clearing enough time to wade in and do it.

**Comment by blakemcbride** [comment]
This has been completed.  It is at https://github.com/blakemcbride/oaklisp
